server/cpho/jinja2/benchmarking/manage_benchmarking_data.jinja2

```diff
@@ -10,11 +10,46 @@
 
 {% macro benchmarking_form(form) %}
   <tr class="benchmarking-form">
-    <td>{{ form.oecd_country }}</td>
-    <td>{{ form.value }}</td>
-    <td>{{ form.year }}</td>
-    <td>{{ form.standard_deviation }}</td>
-    <td>{{ form.comparison_to_oecd_avg }}</td>
+    <td>
+      {{ form.oecd_country }}
+      {% if form.errors.oecd_country %}
+        <div class="error-display">
+          {% for error in form.oecd_country.errors %}<strong>{{ error|escape }}</strong>{% endfor %}
+        </div>
+      {% endif %}
```

Comment:
This is often repeated, can we put the entire block including the conditional in its own macro? e.g. `{{ benchmarking_form_errors(form.oecd_country) }}` it can be defined above this macro